apps/src/templates/projects/projectsRedux.js

```diff
@@ -135,6 +140,22 @@ function selectedGallery(state = initialSelectedGalleryState, action) {
   }
 }
 
+const initialCaptchaState = {
+  captchaSiteKey: '',
+};
+
+function captcha(state = initialCaptchaState, action) {
```

Comment:
I think you can remove the `...state` part here since you're just replacing the value, like we do in `selectedGallery`

```suggestion
function captcha(state = initialCaptchaState, action) {
  switch (action.type) {
    case SET_CAPTCHA_KEY:
      return action.captchaSiteKey;
    default:
      return state;
  }
}
```